apps/studio/components/interfaces/QueryPerformanceV2/IndexSuggestion.tsx

```diff
@@ -0,0 +1,10 @@
+export const IndexSuggestion = () => {
+  return (
+    <div className="h-full overflow-auto flex flex-col gap-y-8 divide-y">
+      <div className="px-4 flex flex-col gap-y-2">
+        <p className="text-sm">Add new index</p>
+        <p>Hello</p>
```

Comment:
Hello?

Reply:
^ ahhh yup this will be updated in the next PR once this is merged - cause the UI is also feature flagged so not user-facing just yet